ArduCopter/mode_auto.cpp

```diff
@@ -686,10 +688,18 @@ bool ModeAuto::verify_command(const AP_Mission::Mission_Command& cmd)
     case MAV_CMD_DO_CHANGE_SPEED:
     case MAV_CMD_DO_SET_HOME:
     case MAV_CMD_DO_SET_ROI:
+#if MOUNT == ENABLED    
```

Comment:
I think there is no need for these changes in the verify_command method.  If we somehow executed this command we want to return true so that we don't stop subsequent commands from being executed.

Reply:
@rmackay9 note that the default case still sets cmd_complete to true - it just also emits a warning that the mission tried to do something the vehicle can't do.

You can see that by expanding the diff in the github interface - the `default:` case set the flag to true.